pkg/frontend/frontend.go

```diff
@@ -0,0 +1,67 @@
+package frontend
+
+import (
+	"context"
+	"fmt"
+	"html/template"
+	"log"
+	"net/http"
+	"os"
+
+	pb "github.com/sachintiptur/grpc-app/proto"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// EnvData struct to store environment variable name and value
+type EnvData struct {
+	EnvName  string
+	EnvValue string
+}
+
+// getEnvData makes gRPC call to get environment variable information
+func (data *EnvData) getEnvData() (err error) {
+	grpcServer := os.Getenv("GRPC_SERVER")
```

Comment:
since the listen port of the gRPC server is not configurable. You have to tread this differently here, this construct would easily allow for misconfiguration, e.g. by setting GRPC_SERVER=1.2.3.4:9999. The server may run on 1.2.3.4 but listen to a different port.
You have different options:
* only accept IPs for GRPC_SERVER (then maybe rename it)
* make the port also configurable on server side.

In both cases you should document it.

Reply:
Here I am assuming user sets GRPC_SERVER as IP:PORT, not just IP. If it is not set, default value of ":9000" is set. I will document this properly.

Reply:
I understand that but on the other side (the server) the port is fix. So you can configure our gRPC client here to connect to a port which the server actually will never be able to listen to, cf. cmd/backend/main.go:15. So whenever u pass in some address with a port other than 9000, the wohle thing will not work even if the IP is right.

Reply:
Fixed. Made GRPC_SERVER configurable on both sides.